master/internal/core_experiment.go

```diff
@@ -417,6 +417,7 @@ func (m *Master) parseCreateExperiment(params *CreateExperimentParams) (
 	dbExp, err := model.NewExperiment(
 		config, modelBytes, params.ParentID, params.Archived,
 		params.GitRemote, params.GitCommit, params.GitCommitter, params.GitCommitDate)
+	dbExp.OriginalConfig = params.ConfigBytes
```

Comment:
why not have NewExperiment handle this instead?

Reply:
good call. I was assuming it is being used elsewhere without direct user config input but it is not being used that way.

Reply:
I updated it